md/comparison-of-7-x-and-6-x.md

```diff
@@ -18,27 +18,27 @@ See the [Installation overview](bonita-bpm-installation-overview.md).
 | Live update  | Not in 6.x  | A live update is a change that is applied to a process or application on the live system, without needing to pause the platform or take a process out of service. See [Live update](live-update.md)  |
 | Form mapping  | Not in 6.x  | A form is mapped to process instantiation, to case overview, or to a task. Any form that respects the relevant contract can be used. This separation between the process and the forms makes it possible to to the live update feature  |
 | Business data  | A process or application can use business data managed by the Bonita Platform. (Subscription editions only)  | Now available in all editions. See [BDM](define-and-deploy-the-bdm.md)  |
-| Actor mapping  | Each actor in a process can be mapped to a Group, Role, Membership or User within an organization. The mapping can be refined using an actor filter.  | No change. See [Actor mapping](actors.md) and [Actor filtering](actor-filtering.md)  |
+| Actor mapping  | Each actor in a process can be mapped to a Group, Role, Membership or User within an organization. The mapping can be refined using an actor filter.  | No change. Anonymous users are not supported anymore. See [Actor mapping](actors.md) and [Actor filtering](actor-filtering.md)  |
```

Comment:
We don't support it from 7.2, and the "support" since 7.0 was more of a hack than a feature.
The use of anonymous users was seen as bad practice at this point, hence why no effort was put to maintain it. Anyway right now it is history/a new PF as far as the product goes, and considering the purpose of this doc page, given it's one of the few point we don't support from 6.x, I feel repeating it on related subject is not too bad.

Reply:
Fine by me.